x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -78,19 +84,14 @@ export const FormContext: React.FC<Props> = ({
 
         // add attachments to the case
         if (updatedCase && Array.isArray(attachments)) {
```

Comment:
not related to this PR but I noticed a validation is missing here for `attachments.length > 0`. could you please add it?